draft-looker-oauth-jwt-cwt-status-list.md

```diff
@@ -122,62 +105,7 @@ Each status of a Referenced Token MUST be represented with a bit size of 1,2,4,
 
 3. The result of the gZIP compression is then base64url-encoded, as defined in Section 2 of {{RFC7515}}.
```

Comment:
don't think base64url-encoding makes sense here now that this part 4.1 is only describing the list independent of JSON/CBOR

Reply:
you are opting to have the base encoding of status list as binary and add the base64 encoding only for JSON? That was my first opinion as well! We were unsure if it would be better to keep things simpler and more common between JSON and CBOR

Reply:
> you are opting to have the base encoding of status list as binary and add the base64 encoding only for JSON? 

That is kinda what I was thinking. I mean, it doesn't seem quite right to base64 encode this binary thing and then place that in CBOR. 

I'd also been thinking that the base encoding of status list as binary could also be how an unsigned list is retrieved via HTTP - i.e. just content type of application/octet-stream - but the bit size also needs to be conveyed there, which complicates that. It would be nice to avoid the size overhead of base64 when not needed though. And it would seem that it's only needed in JSON.

Reply:
The overhead is minimal size wise , if you apply http gzip on top, but it's not "nice"

Reply:
@paulbastian  @c2bo agreed. Separate issue opened #69